components/cam/tools/mkatmsrffile/Makefile

```diff
@@ -1,11 +1,33 @@
-include $(CASEROOT)/Macros.bluefire
+#-------------------------------------
+# Makefile for mkatmsrffile tool 
+#-------------------------------------
 
-LIBDIR=/ptmp/jedwards/SMS_D.f09_g16.B1850CN.bluefire.C.083815/lib
-LIBS = -L$(LIBDIR) -lcsm_share -lpio -lmct -lmpeu -L$(LIB_NETCDF) -lnetcdf -L$(LIB_PNETCDF) -lpnetcdf
-MODDIR=$(LIBDIR)/include
+#-------------------------------------
+#Instructions:
+# Load a compiler and the associated 
+# netcdf library. Update variables "FC"
+# and "NETCDF_PATH" to add the compiler
+# and path to netcdf home directory
+#-------------------------------------
 
+FC := ifort
```

Comment:
You might add a check here to see if FC is already an environment variable, which would allow compiling with something other than ifort (i.e., gnu) without changing anything in the Makefile. You can accomplish this with:
```
ifeq ($(FC),$(null))
    FC = ifort
endif
```